_assets/stylesheets/_card.scss

```diff
@@ -0,0 +1,3 @@
+.card .card-clear { border-style: none; }
```

Comment:
I believe this should be `.card.card-clear` (with no space between) otherwise it's a descendant selector.